src/utils/graph/solver.js

```diff
@@ -0,0 +1,155 @@
+/** @license kiwi.js v1.1.2
```

Comment:
Kiwi.js is only used here?

Reply:
Yeah this is the only file it is used in, just inside the function `solveStrict` below.